.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: confirm, is:bug
+assignees: ''
+
+---
+
+<!--
+Thanks for filing a bug report with NUnit! NUnit is run entirely by volunteers from the community, so we welcome pull requests and contributions in all areas.
+
+To help the others diagnose and reproduce your issue, please include as many of the following as relevant:
+- The full command line being used
+- Method of installation (e.g. NuGet, zip, msi)
+- Log files (Run the console with `--trace=Debug` to produce these)
```

Comment:
Should we also mention the possibility to run `in-process` to diagnose problems?